### PR TITLE
[Task#68465] Add breeze starter kit and Inertia React js to project

### PR DESCRIPTION
**Pull request by tungbd-0214**
## Related Tickets:
- [#68465](https://edu-redmine.sun-asterisk.vn/issues/68465)
- [#68049](https://edu-redmine.sun-asterisk.vn/issues/68049)
- [#68052](https://edu-redmine.sun-asterisk.vn/issues/68052)
- [#68087](https://edu-redmine.sun-asterisk.vn/issues/68087)

## Notes Impacted Areas:
- N/A.

## Attachments:

Screen Shot 2023-09-13 at 10 04 03

Screen Shot 2023-09-13 at 10 04 44

Screen Shot 2023-09-13 at 10 05 08

Screen Shot 2023-09-13 at 10 05 29

### CODE REVIEW

**Review by Hungnd-0370 — APPROVED** [review]
oke

**Comment by tungbd-0214** [comment]
ready

File: .sun-ci.yml [hunk 1]
```diff
@@ -5,68 +5,82 @@ stages:
...
+        paths:
+          - vendor
+    before_script:
+      - cp .env.example .env.testing
+      - composer install
+      - php artisan key:generate --env=testing
+      - php artisan migrate
+      - php artisan config:cache
+      - php artisan config:clear
+      - php artisan cache:clear
```
Comment (ghost) [inline .sun-ci.yml:32]: file sun.ci ko edit indent nha.

File: .sun-ci.yml [hunk 2]
```diff
@@ -5,68 +5,82 @@ stages:
...
+      - php artisan migrate --force --env=testing
+    script:
+      - vendor/bin/phpunit --coverage-clover ./coverage.xml --coverage-html=coverage
+    coverage:
+      type: clover
+      path: coverage.xml
+    artifacts:
+      paths:
+        - coverage
+      expires_in: 3 days
```
Comment (ghost) [inline .sun-ci.yml]: eof

**Comment by tungbd-0214** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**